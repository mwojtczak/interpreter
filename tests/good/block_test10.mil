function main () {
	x = 10;
	print(x);
	y = 42;
	print(y);
	{
		
		x = 5;
		print(x);
		y = 13;
		print(y);
	}
	print(x);
	print(y);
}
