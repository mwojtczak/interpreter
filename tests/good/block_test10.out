10
42
5
13
5
13
